src/tests/multihost/ad/test_adparameters.py

```diff
@@ -475,29 +475,26 @@ def test_0013_bz1794016(self, multihost, adjoin):
         :id: ac5e99cc-2d81-48f7-82ff-622f1c6b3684
         """
         adjoin(membersw='adcli')
-        backup = 'cp -f /etc/sssd/sssd.conf /etc/sssd/sssd.conf.orig'
-        restore = 'cp -f /etc/sssd/sssd.conf.orig /etc/sssd/sssd.conf'
-        multihost.client[0].run_command(backup)
         client = sssdTools(multihost.client[0])
+        client.backup_sssd_conf()
+
         domain_sec_name = client.get_domain_section_name()
-        dom_section = 'domain/%s' % domain_sec_name
-        sssd_params = {'domains': 'files, %s' % domain_sec_name}
+        sssd_params = {'domains': f'local, {domain_sec_name}'}
         client.sssd_conf('sssd', sssd_params)
         domain_params = {'entry_cache_timeout': '5400',
                          'refresh_expired_interval': '4000'}
-        client.sssd_conf(dom_section, domain_params)
-        file_section = 'domain/files'
-        file_params = {'id_provider': 'files'}
-        client.sssd_conf(file_section, file_params)
+        client.sssd_conf(f'domain/{domain_sec_name}', domain_params)
+        local_params = {'proxy_lib_name': 'files',
+                        'passwd_files': '/etc/passwd',
```

Comment:
This is a config option of 'id_provider = files'.

For 'proxy' it should be
```
proxy_lib_name = files
```

See https://sssd.io/docs/files-provider-deprecation.html

Reply:
Done

Reply:
`id_provider = proxy` was added to a wrong domain section (`{domain_sec_name}` instead of `'domain/local'`)

Reply:
Moved to domain/local